### DISCUSSION TITLE
这...

### DISCUSSION THREAD

**Discussion by lkadsjcnska** [WorldBoxOpenMods/ModLoader#28] [General]
不是，咋还有破百度网盘，一看就是国人😂😂

**Comment by lkadsjcnska** [comment]
Foreigners said:
???
Besides, Baidu is Chinese, so they can't get in

**Reply by lkadsjcnska** [reply]
chinese
外国人表示：
？？？
况且百度是中国的，他们也进不去吧

**Comment by inmny** [comment]
百度网盘那个不是在"中文"的介绍里吗？英文的那个用的谷歌（之前貌似给人举报挂了，懒得恢复了）
